src/lando/utils/management/commands/setup_dev.py

```diff
@@ -25,7 +25,7 @@ def setup_workers(self):
                 self.stdout.write(f"Found {worker} worker.")
             except Worker.DoesNotExist:
                 # Set the name of the worker to match the SCM.
-                worker = Worker(name=worker_scm)
+                worker = Worker(name=worker_scm, scm=worker_scm)
```

Comment:
Would be good to add a bullet in the commit message to mention this change.

Reply:
Yep, that was a separate bug that stood in the way.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1938844 #186